packages/react-table/src/components/Table/examples/Table.md

```diff
@@ -1812,6 +1816,171 @@ class EditableRowsTable extends React.Component {
+          disableSelection: true
+        },
+        {
+          favorited: true,
+          cells: ['b', 'two', 'b', 'four', 'five']
+        }
+      ],
+      sortBy: {},
+      canSortFavorites: true
+    };
+    this.onSelect = this.onSelect.bind(this);
+    this.onFavorite = this.onFavorite.bind(this);
+    this.onSort = this.onSort.bind(this);
+    this.toggleFavsSort = this.toggleFavsSort.bind(this);
+  }
+
+  onSelect(event, isSelected, rowId) {
+    let rows;
+    if (rowId === -1) {
+      // header row
+      rows = this.state.rows.map(oneRow => {
+        oneRow.selected = isSelected;
+        return oneRow;
+      });
+    } else {
+      // body row
+      rows = [...this.state.rows];
+      rows[rowId].selected = isSelected;
+    }
+    this.setState({
+      rows
+    });
+  }
+
+  onFavorite(event, isFavorited, rowId) {
+    this.setState({
+      rows: this.state.rows.map((row, index) => {
+        if (index === rowId) {
+          row.favorited = isFavorited;
+          row.favoritesProps = {
+            ...row.favoritesProps,
+            // Example of how to override the default aria-label of Starred / Not starred
+            'aria-label': isFavorited ? 'Favorited' : 'Not favorited'
+          };
+        }
+        return row;
+      })
+    });
+  }
+
+  onSort(_event, index, direction) {
+    // index 0 === select column
+    // index 1 === favorites column
+    // index 2 === first user defined column
+    let sortedRows;
+    if (index === 1) {
+      // favorites column
+      sortedRows = this.state.rows.sort((a, b) => {
+        if (a.favorited && !b.favorited) {
+          return -1;
+        } else if (!a.favorited && b.favorited) {
+          return 1;
+        }
+        return 0;
+      });
+    } else {
+      const userIndex = index - 2; // subtract for select and favorites column
+      sortedRows = this.state.rows.sort((a, b) =>
+        a.cells[userIndex] < b.cells[userIndex] ? -1 : a.cells[userIndex] > b.cells[userIndex] ? 1 : 0
+      );
+    }
+    this.setState({
+      sortBy: {
+        index,
+        direction
+      },
+      rows: direction === SortByDirection.asc ? sortedRows : sortedRows.reverse()
+    });
+  }
+
+  toggleFavsSort(checked) {
+    this.setState({
+      canSortFavorites: checked
+    });
+  }
+
+  render() {
+    const { columns, rows, sortBy, canSortFavorites } = this.state;
+
+    return (
+      <div>
+        <Checkbox
+          label="Can sort favorites"
+          isChecked={canSortFavorites}
+          onChange={this.toggleFavsSort}
+          aria-label="toggle select all checkbox"
+          id="toggle-select-all"
+          name="toggle-select-all"
+        />
+        <Table
+          // using this prop enables the favorites column
+          onFavorite={this.onFavorite}
+          // if the onSort prop is detected, favorites can be sorted
+          // if you want to exclude favorites from sorting you can use this prop with a value of `false`
```

Comment:
These comments are great.  I wonder if it makes sense to add them to the prop docs?

Reply:
yes, I added a description for the Favorites example that explains how to enable favoriting, and in the props I added
```
/**
   * Enables favorites column
   * Callback triggered when a row is favorited/unfavorited
   */
  onFavorite?: OnFavorite;
  /** Along with the onSort prop, enables favorites sorting, defaults to true */
  canSortFavorites?: boolean;
```